mmeds/config.py

```diff
@@ -214,7 +222,10 @@
 MODULE_ROOT = DATABASE_DIR.parent / '.modules/modulefiles'
 
 if not DATABASE_DIR.exists():
-    DATABASE_DIR.mkdir()
+    try:
+        DATABASE_DIR.mkdir()
+    except FileExistsError:
+        pass
```

Comment:
should at least log the exception I would think

Reply:
I don't think it's necessary. It's just for creating those directories the first time that the config is loaded on a particular machine. Logging that those files exist for every subsequent loading of this file is going to clog up the logs with a lot of unhelpful information

Reply:
fair enough